.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,4 +17,4 @@ spellcheck: fixed a few typos
 experiment: added an experimental thingy
 /:cl:
 
-[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:) 
+[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
```

Comment:
Why two lines?

Reply:
Just to make it foolproof. With how lines wrap, sometimes people think they're writing on a new line when really they're not. When that happens, the comment breaks.